mkinitcpio.d/example.preset

```diff
@@ -8,6 +8,7 @@ PRESETS=('default' 'fallback')
 # as the path to an kernel image.
 ALL_kver='/boot/vmlinuz-linux'
 ALL_config='/etc/mkinitcpio.conf'
+ALL_microcode="/boot/intel-ucode.img"
```

Comment:
In the real presets, will this somehow autodetect `amd-ucode.img`?

Reply:
I'm not implementing cross-distro autodetection of microcode updates.

We can make this and array and cat together every file it finds. But I think it's better to let the user decide.

Reply:
Then maybe it's better to make it an array, e.g. `(/boot/*-ucode.img)` by default. So far users haven't had to edit presets by default, I'm not really sure if it's a good idea to add yet another place that must be changed during installation.

Reply:
I don't think having UEFI executable generation on by *default* is the goal. This is going to be optional. Manual configuration for that purpose is fine.

Reply:
But also, why make it harder when the array option doesn't seem like a significant amount of work to include?
If it's purely an effort kinda deal, I'd be up for fixing this in a follow-up patch.

Reply:
That wasn't an argument against using arrays. It was more about supplying default configurations or not.

Reply:
This was painfull, but implemented!